department-of-veterans-affairs/caseflow
# Resolve interaction delays for Test Users dropdown

### Description
Resolve interaction delays for test users dropdown 

### Acceptance Criteria
- [ ] Can select and type test user without delay after clicking into dropdown 

### Testing Plan
1. Go to `/test/users`
2. Click into dropdown
3. Verify options load without delay 
3. Begin typing user name
4. Verify that options load without delay

### User Facing Changes
